plugins/@grouparoo/salesforce/src/lib/export/options.ts

```diff
@@ -30,19 +30,17 @@ export const getDestinationOptions: SalesforceDestinationOptions = async ({
   const cacheData: SalesforceCacheData = { appId, appOptions };
   const out: DestinationOptionsMethodResponse = {};
   const conn = await connect(appOptions);
-  fieldsOptions.map(async (fieldsOption) => {
-    Object.assign(
-      out,
-      await getOptions(
-        conn,
-        cacheData,
-        destinationOptions,
-        fieldsOption.fieldName,
-        fieldsOption.fieldValue,
-        fieldsOption.specialFields
-      )
+  for (const fieldsOption of fieldsOptions) {
```

Comment:
hmm, more dangerous than I'd like to see: try `fieldOption` here and in the loop.